app/scripts/models/notifications.js

```diff
@@ -51,22 +53,20 @@ define([
       });
     },
 
-    profileUpdated: function (data) {
-      this.triggerAll(EVENTS.PROFILE_CHANGE, data);
-    },
-
-    accountDeleted: function (data) {
-      this.triggerAll(EVENTS.DELETE, data);
-    },
-
     // Listen for notifications from other fxa tabs or frames
     _listen: function (tabChannel) {
       var self = this;
       _.each(EVENTS, function (name) {
         tabChannel.on(name, self.trigger.bind(self, name));
       });
+    },
+
+    clear: function () {
+      if (this._tabChannel) {
+        this._tabChannel.clear();
+      }
     }
-  }, Backbone.Events);
+  }, EVENTS);
 
   return Notifications;
```

Comment:
Should the file name and this constructor function also be renamed, given that everywhere else now refers to instances as `notifier` rather than `notifications`?